react/features/invite/components/info-dialog/web/InfoDialogButton.js

```diff
@@ -47,10 +47,10 @@ type Props = {
     _liveStreamViewURL: ?string,
 
     /**
-     * The number of real participants in the call. If in a lonely call, the
+     * True if the number of real participants in the call is less than 2. If in a lonely call, the
      * {@code InfoDialog} will be automatically shown.
      */
-    _participantCount: number,
+    _participantCountLessThan2: boolean,
```

Comment:
Maybe we should call this `_lonelyCall` instead then? We could also encapsulate the check in some `isLonleyCall` function, IIRC mobile also performs this check.

Reply:
I'll rename it to _isLonelyCall

Reply:
https://github.com/jitsi/jitsi-meet/blob/fb8ef366c673c357183d5e568aaf127428c320ae/react/features/filmstrip/components/native/Thumbnail.js#L259

https://github.com/jitsi/jitsi-meet/blob/fb8ef366c673c357183d5e568aaf127428c320ae/react/features/toolbox/functions.native.js#L17

Reply:
ok. missed the second one. but the first one is practically not a lonely call but a p2p call, right?